charts/newrelic-infrastructure-v3/templates/_helpers.tpl

```diff
@@ -188,3 +196,121 @@ Returns Custom Attributes even if formatted as a json string
 {{- define "newrelic.customAttributes" -}}
 {{- merge (include "newrelic.customAttributesWithoutClusterName" . | fromJson) (dict "clusterName" (include "newrelic.cluster" .)) | toJson }}
 {{- end -}}
+
+
+{{/*
+Returns controlPlane component taking into account legacy values
+*/}}
```

Comment:
Helm does not template [anything starting with `_`](https://helm.sh/docs/chart_template_guide/named_templates/#partials-and-_-files) so it could be a good idea to separate the helpers into different files (or the compatibility layer to other file so it could be removed more easily in the future.

Reply:
sure, makes sense!